Content.Server/NukeOps/Components/WarDeclaratorComponent.cs

```diff
@@ -0,0 +1,42 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.NukeOps
+{
+    [RegisterComponent]
+    public sealed class WarDeclaratorComponent : Component
+    {
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("message")]
+        public string Message { get; set; } = string.Empty;
+        
+        /// <summary>
+        /// Fluent ID for default message
+        /// </summary>
+        [DataField("defaultMessage")]
+        public string DefaultMessage { get; set; } = string.Empty;
+
```

Comment:
Redundant fields?

Reply:
Are they? "message" is current message in text field, "defaultMessage" is default localized message that used on component initialize. I can simply put some text in message but then it will be unlocalized.

Reply:
Can't Message just use the default localised field?